### PR TITLE
Add Mailjet destination page, update logo, and what's new post

### PR DESCRIPTION
**Pull request by edmundito**
## Change description

This makes a few changes to add more information about Mailjet:
- What's New Post
- Logo update (the logo has changed since its release)
- Destination info page

### Development

- [x] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/whats-new/2021-09-01-mailjet.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Mailjet Destination"
+date: "2021-09-01"
```
Comment (pedroslopez) [inline pages/whats-new/2021-09-01-mailjet.mdx:3]: Assuming this date goes back to when this was released?

File: data/plugins/mailjet.ts [hunk 2]
```diff
@@ -11,6 +11,42 @@ export const Mailjet: Plugin = {
...
+  },
+  useCasesList: [
+    "Automatically create Contacts as new users sign up in your core product",
+    "Trigger email campaigns based on dynamic Audiences",
+  ],
+  dataModelTitle: "An Overview of Mailjet’s Data Model and API",
+  dataModelDescription: [
+    "Trying to figure out how to integrate with Mailjet? Here’s an overview for how Mailjet defines customers.",
+    "The core object in Mailjet is a Contact. A Contact represents an individual person that lives on a Contact list. Contacts have one required attribute, which is their email address.",
+    "Mailjet calls attributes of a Contact a Contact Property. Mailjet pre-defines a number of Contact Properties such as <code>firstname</code> and <code>country</code>, but you can define as many Contact Properties as you'd like in the Mailjet app.",
```
Comment (pedroslopez) [inline data/plugins/mailjet.ts]: Using `app` might imply a Grouparoo app - maybe `Mailjet website` or just `in Mailjet`?

**Review by pedroslopez — COMMENTED** [review]
Looks good, just a question on the date and small suggestion.

File: pages/whats-new/2021-09-01-mailjet.mdx (hunk 1 above)
Comment (edmundito) [reply pages/whats-new/2021-09-01-mailjet.mdx:3]: Yes, I didn't know when exactly to introduce this, but v6 was released around the end of october.

Comment (edmundito) [reply pages/whats-new/2021-09-01-mailjet.mdx:3]: https://github.com/grouparoo/grouparoo/releases/tag/v0.6.0

File: pages/whats-new/2021-09-01-mailjet.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+tags: ["new"]
+description: "Grouparoo can now use Mailjet as a Destination."
+github: "https://github.com/grouparoo/grouparoo/pull/2202"
+---
+
+With the release of Grouparoo v0.6, you can sync all of your customer data that lives
+in your data warehouse into [Mailjet](https://www.mailjet.com/) with the `@grouparoo/mailjet` plugin.
+
+Learn more about integrating with Grouparoo with Mailjet [here](/integrations/destinations/mailjet).
```
Comment (pedroslopez) [inline pages/whats-new/2021-09-01-mailjet.mdx]:
```suggestion
Learn more about integrating Grouparoo with Mailjet [here](/integrations/destinations/mailjet).
```

**Review by pedroslopez — CHANGES_REQUESTED** [review]
One more small change and we should be good to go!

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**